### PR TITLE
On Boarding Catalog Item Automation

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed

File: src/test/java/PLATFORM_BUSINESS/NERD/selenium/StepsImplementation/NERDApplicationStepsImplementation.java [hunk 1]
```diff
@@ -268,7 +268,7 @@ public static void checkingEmailWasNotReceived(String docPlanningContact) {
     public static void userIsOnSubmissionsPage(String applicationName) {
         WebDriverUtils.webDriver.get(getNerdUrl());
         WebDriverUtils.webDriver.navigate().refresh();
-        CommonUtils.sleep(5000);
+        CommonUtils.sleep(55000);
```
Comment (hamidnesar) [inline src/test/java/PLATFORM_BUSINESS/NERD/selenium/StepsImplementation/NERDApplicationStepsImplementation.java:271]: Are you pausing the test run for 55 seconds?

File: src/test/java/PLATFORM_BUSINESS/NERD/selenium/StepsImplementation/NERD_CRS_ReviewersStepImplementation.java [hunk 2]
```diff
@@ -167,7 +167,7 @@ public static void theUserIsRedirectedToTheKnowledgeBaseViewPage() {
      * @param topAccomplishmentsAccordionText The text of the header label for the accordion.
      */
     public static void thereIsACollapsedAccordionWithRheHeaderLabeled(String topAccomplishmentsAccordionText) {
-        CommonUtils.sleep(20000); //THIS SLEEP TIME IS NEEDED FOR THE NERD PAGE TO LOAD THE RECORDS
+        CommonUtils.sleep(30000); //THIS SLEEP TIME IS NEEDED FOR THE NERD PAGE TO LOAD THE RECORDS
```
Comment (hamidnesar) [inline src/test/java/PLATFORM_BUSINESS/NERD/selenium/StepsImplementation/NERD_CRS_ReviewersStepImplementation.java:170]: I would suggest using dynamic waits if possible

**Review by hamidnesar — COMMENTED** [review]
Hi, 
Please review the comments. Thanks!

**Review by hamidnesar — APPROVED** [review]
Approved!

**Review by sonika-nih — APPROVED** [review]

**Review by alenaminsk — APPROVED** [review]

**Pull request merged**